.github/workflows/ci-e2e-badger.yaml

```diff
@@ -41,6 +43,12 @@ jobs:
             ;;
         esac
 
+    - uses: ./.github/actions/verify-metrics-snapshot
+      if: matrix.version == 'v2'
+      with:
+        snapshot: badger_metrics.txt
```

Comment:
nit: let's change all snapshots to be of the form `metrics_snapshot_${storage}` so that they are grouped together by name